CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## tip
 
+* BUGFIX: correct the queries for `Label Filters` and `Metrics Browser` for metrics with special characters. See [this issue](https://github.com/VictoriaMetrics/grafana-datasource/issues/140)
```

Comment:
```suggestion
* BUGFIX: allow the display of metrics with special characters in `Label Filters` and `Metrics Browser`. See [this issue](https://github.com/VictoriaMetrics/grafana-datasource/issues/140)
```

Reply:
I'm not sure if this description is accurate. The issue was that when attempting to retrieve `label=values` for metrics with special characters, the characters were not properly escaped, resulting in the query returning an error.